docs/examples.rst

```diff
@@ -0,0 +1,7 @@
+Examples
+========
+
+.. toctree::
+    :maxdepth: 1
+
+    examples/test-example
```

Comment:
Note to self: #7 replaces this with `configuration` which then becomes a part of the PyPi release.